src/Compilers/CSharp/Test/Semantic/Diagnostics/GetDiagnosticsTests.cs

```diff
@@ -394,5 +394,43 @@ public void CompilingCodeWithSameParseOptionsInMultipleSyntaxTreesShouldReportOn
             Assert.True(diagnostics[0].Location.SourceTree.Equals(syntaxTree1));
             Assert.True(diagnostics[1].Location.SourceTree.Equals(syntaxTree2));
         }
+
+        [Fact]
+        [WorkItem(11497, "https://github.com/dotnet/roslyn/issues/11497")]
+        public void ConsistentErrorMessageWhenProvidingNullKeyFile()
```

Comment:
The issue/PR has nothing to do with signing. It is just about input validation, and whether we're consistent when reporting error strings or not.
I'll edit these tests to just verify CompilationOptions.Errors in that case.
Please let me know if you disagree.

---
In reply to: [109721642](https://github.com/dotnet/roslyn/pull/18407#discussion_r109721642) [](ancestors = 109721642)